lib/srv/term_test.go

```diff
@@ -74,10 +72,10 @@ func (s *TermSuite) TestGetOwner(c *check.C) {
 
 	for _, tt := range tests {
 		uid, gid, mode, err := getOwner("", tt.inUserLookup, tt.inGroupLookup)
-		c.Assert(err, check.IsNil)
+		require.NoError(t, err)
 
-		c.Assert(uid, check.Equals, tt.outUID)
-		c.Assert(gid, check.Equals, tt.outGID)
-		c.Assert(mode, check.Equals, tt.outMode)
+		require.Equal(t, uid, tt.outUID)
```

Comment:
Technically expected value should come before actual value in the assertion here.